src/main/kotlin/io/emeraldpay/dshackle/Config.kt

```diff
@@ -16,14 +16,7 @@
  */
 package io.emeraldpay.dshackle
 
-import io.emeraldpay.dshackle.config.CacheConfig
-import io.emeraldpay.dshackle.config.HealthConfig
-import io.emeraldpay.dshackle.config.MainConfig
-import io.emeraldpay.dshackle.config.MainConfigReader
-import io.emeraldpay.dshackle.config.MonitoringConfig
-import io.emeraldpay.dshackle.config.SignatureConfig
-import io.emeraldpay.dshackle.config.TokensConfig
-import io.emeraldpay.dshackle.config.UpstreamsConfig
+import io.emeraldpay.dshackle.config.*
```

Comment:
Maybe return to specific imports?

Reply:
I'm going to make it in separate PR because we need to fix that behavior in editor config and reforamat all files (it